app.js

```diff
@@ -8,6 +8,23 @@ app.get('/api/topics', getAllTopics)
 
 app.get('/api', getAllEndpoints)
 
+app.get('/api/articles/:article_id', getArticleById)
+
+app.use((err, req, res, next) => {
+    console.log(err)
+    if(err.code === '22P02'){
+      res.status(400).send({msg: 'Bad request'})
+    }
+    next(err)
+  });
+  
+  app.use((err, req, res, next) => {
```

Comment:
There error handling middleware functions could be extracted into their own file if you'd like to streamline your app.js